### ISSUE TITLE
9 - Deep Learning Basics

### ISSUE THREAD

**Issue by CSFelix** [CSFelix/data-science-studies#26]
# ✔️ Tasks

- ✔️ Basic Keras and Tensor Flow
- ✔️ How to Display Activation Functions Plots
- ✔️ Underfitting and Overfitting
- ✔️ Dropout and Batch Normalization Layers
- ✔️ Binary Classification
- ✔️ LSTM and GRU
